### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes various typos I spotted in the project.

### CODE REVIEW

**Review by flimzy — APPROVED** [review]

File: pouchdb/find_test.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func TestExplain(t *testing.T) {
 	tests.Add("query error", test{
 		db:    &db{db: bindings.GlobalPouchDB().New("foo", nil)},
 		query: nil,
-		err:   "TypeError: Cannot read propert",
+		err:   "TypeError: Cannot read property",
```
Comment (flimzy) [inline pouchdb/find_test.go:83]: This one was actually correct before, as it's a regular expression, meant to match on `property` and `properties`, I believe.  It seems that at present, with the modern versions of NodeJS and PouchDB, it's always `properties`, though.

```suggestion
		err:   "TypeError: Cannot read properties",
```

**Review by flimzy — CHANGES_REQUESTED** [review]
Thanks for the PR.

Just one minor change needed to get the tests passing again.

File: pouchdb/find_test.go (hunk 1 above)
Comment (deining) [reply pouchdb/find_test.go]: Thnaks for the clarification, changed as requested.

**Comment by deining** [comment]
> Thanks for the PR.

You are welcome.
 
> Just one minor change needed to get the tests passing again.

Done. Looking forward seeing ths PR merged.

**Review by flimzy — APPROVED** [review]

**Pull request merged**